Tests/SwiftGenKitTests/TestsHelper.swift

```diff
@@ -136,7 +136,7 @@ func XCTDiffContexts(_ result: [String: Any],
   if ProcessInfo().environment["GENERATE_CONTEXTS"] == "YES" {
     let target = Path(#file).parent().parent() + "Fixtures/StencilContexts" + directory.rawValue + fileName
     do {
-      let data = try Yams.dump(object: result)
+      let data = try Yams.serialize(node: YamsFix.represent(result))
```

Comment:
Instead of just extracting the represent fix in a dedicated enum maybe we could have a tiny wrapper around Yams — would especially be useful when we'll need it in the main target later to allow end users to output YAML from SwiftGen (eg the YAML/JSON PR)?

Like:
```swift
enum YAML {
  func decode(data: Data) throws -> Any {
    // Simple forwarding for now, but maybe
    // later well use a custom Construc
    // to parse only `true`/`false` as Booleans
    // And/or to parse env variables like SwiftLint does etc
    return Yams.load(data: data, .basiit
  }
  func encode(object: Any) throws -> Data {
    return Yams.serialize(fixedRepresent(object))
  }
  private fixedRepresent(…) …
}
```

I know this is intended to only be a temporary fix but extracting the  whole YAML load/dump in a dedicated type in SwiftGen will make it easier to generalize  and uniformize our YAML parsing/dumping configurations/rules across SwiftGen, both for the test contexts, for the config file (not much affected by this dumping fix but still) and future PRs.

Reply:
And that would make changing the fix to the official solution (once implemented in YAMS) easier to do as all load/dump code invoking YAMS will be in one place 😉

Reply:
Where should the file with this code go? (which target?)

Reply:
Probably SwiftGenKit?